docs/py-modindex.html

```diff
@@ -0,0 +1,286 @@
+<!DOCTYPE html>
```

Comment:
is this html file needed?

Reply:
Yes this file was generated when building the docs and was missed in the last PR.